### DISCUSSION TITLE
NO ISSUE JUST WANT CLARAFICATION

### DISCUSSION THREAD

**Discussion by Rolozity** [gucci-on-fleek/lockdown-browser#89] [Q&A]
After debugging and using ChatGPT for the last 32 hours...I finally got it to work...

However, I'm scared to use it on the actual exam.  My alt+tab switching was completely undetected, and I could sign into my account and everything.

For the people who said it works, DOES IT still work?   And for the people who got it to work in practice, did it mess up on the actual exam?  I need more answers before taking the risk.

**Comment by Voidless7125** [comment]
This does work and is *mostly* undetectable. Just don't change the window size, and you will be fine.  (If you have your camera on, switch Windows, and start typing, it might be obvious that you're not typing in lockdown.)

Also, you needed to install Visual Studio with W SDK and MSVC C++ and git for it to work =)
(Tested on 2.1.2.10 and it works.)

**Comment by OracleOfWisdom** [comment]
@Voidless7125 Could you elaborate on what Tested means? Do you have a test quiz environment where you are the admin?

**Comment by Voidless7125** [comment]
> [@Voidless7125](https://github.com/Voidless7125) Could you elaborate on what Tested means? Do you have a test quiz environment where you are the admin?

I tested as I loaded a test, switched back and forth, and completed the test in the sandbox.
And no, I did not use a practice test, I used a real test.

**Comment by Encantbr** [comment]
I just tried to do an exam and received this notification, on windows, version 2.1.2.10

LockDown Browser
Warning: You are in violation of the End User License Agreement
LockDown Browser's End User License Agreement states: "You may not use LockDown Browser in any way which attempts to bypass the locked browser environment or subvert the purpose of the software, including but not limited to using LockDown Browser in conjunction with applications or services designed to bypass the locked browser environment."
Warning: If you attempt to bypass it again, you will be banned from using LockDown Browser.
If you have questions, contact the LockDown Browser license administrator at your institution and provide the reference code below. Your institution can open a support ticket at https://support.respondus.com

**Reply by gucci-on-fleek** [reply]
(Moved to #101)
